core/src/test/java/org/infinispan/eviction/impl/ManualEvictionWithPassivationAndSizeBasedAndConcurrentOperationsInBackupOwnerTest.java

```diff
@@ -53,8 +58,7 @@ protected void assertInMemory(Object key, Object value) {
       CacheLoader<Object, Object> loader = TestingUtil.getFirstLoader(cache);
       assertNotNull("Key " + key + " does not exist in data container", entry);
       assertEquals("Wrong value for key " + key + " in data container", value, entry.getValue());
-      MarshallableEntry<Object, Object> entryLoaded = loader.loadEntry(key);
-      assertNull("Key " + key + " exists in cache loader.", entryLoaded);
+      eventually(() -> loader.loadEntry(key) == null);
```

Comment:
Same nitpick, if it's already in memory it shouldn't be in the loader (in the next file as well)

Reply:
That is just how the test was originally. I think it was trying to guarantee that it was also guaranteeing passivation rules of the entry must be in one but not the other.

Reply:
Same thing about async activation.

Reply:
I was complaining about the change from `assertNull` to `eventually`, I was expecting the activation to be done when the operation was done.